execution_engine/src/core/engine_state/genesis.rs

```diff
@@ -540,7 +541,8 @@ impl Distribution<GenesisConfig> for Standard {
 pub struct ExecConfig {
     accounts: Vec<GenesisAccount>,
     wasm_config: WasmConfig,
-    system_config: SystemConfig,
+    core_config: CoreConfig,
+    system_costs: SystemCosts,
```

Comment:
Why are you renaming SystemConfig? I would expect it to retain naming consistency w/ other config injections.

Reply:
I found it confusing that `SystemConfig` was actually referenced under `[system_costs]` in the `chainspec.toml`, so I renamed it to match what it contains and as it is defined in node's `Chainspec` struct:
https://github.com/CasperLabs/casperlabs-node/blob/cbf37d59d2bec1cedefcac769f309a8f6810acab/node/src/types/chainspec.rs#L66-L67. I think the new name makes more sense in this context but I'm open to revisiting this

Reply:
Done